tests/lib/rules/max-classes-per-file.js

```diff
@@ -0,0 +1,75 @@
+/**
+ * @fileoverview Tests for max-classes-per-file rule.
+ * @author James Garbutt <https://github.com/43081j>
+ */
+"use strict";
+
+//------------------------------------------------------------------------------
+// Requirements
+//------------------------------------------------------------------------------
+
+const rule = require("../../../lib/rules/max-classes-per-file"),
+    RuleTester = require("../../../lib/testers/rule-tester");
+
+//------------------------------------------------------------------------------
+// Tests
+//------------------------------------------------------------------------------
+
+const ruleTester = new RuleTester();
```

Comment:
Nitpick: we can use `RuleTester({parserOptions: { ecmaVersion: 6 }})` here, so no longer need parserOptions in every test case.